mobile_verifier/src/coverage.rs

```diff
@@ -218,9 +202,12 @@ impl CoverageDaemon {
         self.coverage_obj_sink.commit().await?;
         transaction.commit().await?;
 
-        // After writing all of the coverage objects, we set their oracle boosting assignments. This is
-        // done in two steps to improve the testability of the assignments.
-        let unassigned_hexes = UnassignedHex::fetch(&self.pool);
+        // After writing all of the coverage objects, we set their oracle boosting assignments.
+        // If the data sets are not ready, we leave everything as NULL.
+        if !self.hex_boost_data.is_ready().await {
+            return Ok(());
```

Comment:
If it's okay to not do anything when boosting data isn't ready, it makes me wonder if the `CoverageDaemon` needs a copy of `HexBoostData` at all.

Reply:
Should we just make the mobile-verifier check that it has one of each requried file type and not start the TaskManager until it has the required files ready?